test/test_data/systemd.py

```diff
@@ -15,12 +15,12 @@
         "product": "systemd",
         "version": "239",
     },
-    {
-        "url": "http://security.ubuntu.com/ubuntu/pool/main/s/systemd/",
-        "package_name": "systemd_229-4ubuntu21.27_amd64.deb",
-        "product": "systemd",
-        "version": "229",
-    },
+    # {
```

Comment:
the Ubuntu package doesn't get detected by the systemd checker and was returning `UNKNOWN` version.

Reply:
resolved in 212f35a1b1961c163245301535ee602b7396bbeb